store/admin/bin/boot

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# This script is designed to be run inside the container
+#
+
+ETCD_PORT=${ETCD_PORT:-4001}
+ETCD="$HOST:$ETCD_PORT"
+
+confd -node $ETCD -config-file /app/confd.toml &
+
+# loop forever until the container is stopped
+while true; do
+  sleep 5
```

Comment:
Would this mean that at the longest, it'd take 5 seconds for the container to die? Would there be an issue if we reduced it to `sleep 1`?